cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -35,12 +35,15 @@ const (
 //PoolAddEventHandled is a flag representing if the pool has been initially imported or created
 var PoolAddEventHandled = false
 
+//It declares a variable string of type string to store the prefix of pool name
```

Comment:
@debck - check if the lint error is fixed with this comment. Doesn't it require you to start the sentence with variable name?